.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Maybe ask someone more well-versed in JS what should exist in the .gitignore. Just .idea feels weird, but I've no idea